ferrocene/doc/internal-procedures/src/release/index.rst

```diff
@@ -24,6 +24,17 @@ to instruct the release process to release. The script does little when
 starting a manual release, but it's fully responsible for determining which
 scheduled releases to run.
 
+Steps to a release
```

Comment:
I think this describes `publish-release` and can be a lesser heading, since it is effectively describing what one of our tools for "Release tooling" does?

Reply:
sure, done